File: backend/src/main/java/com/graphs/entity/RankingResponseV2.java
package com.graphs.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class RankingResponseV2 {
    List<Double> timeRanking;
    List<Double> scoreRanking;
}

File: frontend/src/app/graph/RankingResponseV2.ts
export class RankingResponseV2 {
  timeRanking: number[];
  scoreRanking: number[];

  constructor(timeRanking: number[], scoreRanking: number[]) {
    this.timeRanking = timeRanking;
    this.scoreRanking = scoreRanking;
  }
}
